orb-ui/src/engine/animations/arc_dash.rs

```diff
@@ -63,6 +63,10 @@ impl<const N: usize> ArcDash<N> {
 impl<const N: usize> Animation for ArcDash<N> {
     type Frame = RingFrame<N>;
 
+    fn name(&self) -> &'static str {
```

Comment:
Instead of using a `name` method, why not use a `kind` method that returns an enum that identifies which animation it is?

That enum can then just be printed with Debug.